bip-0085.mediawiki

```diff
@@ -370,6 +416,17 @@ The reason for running the derived key through HMAC-SHA512 and truncating the re
 
 Many thanks to Peter Gray and Christopher Allen for their input, and to Peter for suggesting extra application use cases.
 
+==Change Log==
+
+===1.1.0===
```

Comment:
I think this would need a date, for instance, like in BIP352:

```
== Changelog ==

* '''1.0.1''' (2024-06-22):
** Add steps to fail if private key sum is zero (for sender) or public key sum is point at infinity (for receiver), add corresponding test vectors.
* '''1.0.0''' (2024-05-08):
** Initial version, merged as BIP-352.
```

However, having looked at all the changes in https://github.com/bitcoin/bips/commits/master/bip-0085.mediawiki, I think it's also fine to defer that to a separate PR that adds a fuller history.

Reply:
What I'd like to do for now is date 1.1.0 and leave the rest to backfill.